cvat-ui/src/reducers/interfaces.ts

```diff
@@ -584,3 +584,10 @@ export enum DimensionType {
     DIM_3D = '3d',
     DIM_2D = '2d',
 }
+
+export enum ViewType {
+    PERSPECTIVE = 'perspective',
+    TOP = 'top',
+    SIDE = 'side',
+    FRONT = 'front',
+}
```

Comment:
I think we can import it from canvas-3d module. We should not declare the type again.

Reply:
Resolved